packages/expo/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - Use `typeof window` checks for removing server code. ([#27514](https://github.com/expo/expo/pull/27514) by [@EvanBacon](https://github.com/EvanBacon))
 - [expo-updates] Migrate to requireNativeModule/requireOptionalNativeModule. ([#25648](https://github.com/expo/expo/pull/25648) by [@wschurman](https://github.com/wschurman))
 - Remove implicit dependency on expo-updates to do runtime version check at runtime. ([#26080](https://github.com/expo/expo/pull/26080) by [@wschurman](https://github.com/wschurman))
+- [Android] Added bridgeless support on ReactNativeHostHandler and drop support. Also dropped support for React Native 0.73 and lower. ([#27629](https://github.com/expo/expo/pull/27629) by [@kudo](https://github.com/kudo))
```

Comment:
```suggestion
- [Android] Added bridgeless support on ReactNativeHostHandler and dropped support for React Native 0.73 and lower. ([#27629](https://github.com/expo/expo/pull/27629) by [@kudo](https://github.com/kudo))
```

But maybe it should be added as a separate entry in the breaking changes section?